.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+  pull_request:
+    branches: ['*']
+  push:
+    branches: ['master']
+
+jobs:
+  build:
+    name: Build and Test
+    strategy:
+      matrix:
+        java:
+          - adopt@1.8
+          - adopt@1.11
```

Comment:
Do you need to give the version of scala to use here or will that be pulled in from the build.dbt file?

Reply:
From build.sbt. Also, it's a Java project, though it does use Scala for the unit tests, which is there's no need for a matrix of Scala versions.

Reply:
I should have looked at that.  I just assumed it was scala.  😊